app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,27 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+
+ .green-completion{
+  color: white;
+  width: 85px;
+  background-color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.